tests/test_c8ql.py

```diff
@@ -10,7 +12,54 @@
     C8QLQueryKillError,
     C8QLQueryValidateError,
 )
-from tests.helpers import assert_raises, extract
+from tests.helpers import assert_raises, extract, generate_col_name
+
+
+def test_export_data_query(client, docs):
+    collection_name = generate_col_name()
+    client.create_collection(collection_name)
+    client.insert_document(collection_name=collection_name, document=docs)
+    resp = client.export_data_query(query='FOR u IN @@collection RETURN u',
+                                    bind_vars={'@collection': collection_name})
+    assert resp['error'] is False
+    assert resp['result'] == docs
+
+    random_value = random.randint(1, len(docs))
+    resp = client.export_data_query(query='FOR u IN @@collection FILTER u.val > @value RETURN u',
+                                    bind_vars={'@collection': collection_name, 'value': random_value})
+    assert resp['error'] is False
+    assert len(resp['result']) == (len(docs) - random_value)
+
+    resp = client.export_data_query(query='FOR u IN {} RETURN u'.format(collection_name))
+    assert resp['error'] is False
+    assert resp['result'] == docs
+
+    with assert_raises(C8QLQueryExecuteError) as err:
```

Comment:
you can separate this test into two.. like have a different combined test for these 5 errors.. and for this error test you can add "client" and "col" in params.. you can replace collection_name with "col.name".